src/Tools/dotnet-monitor/HostBuilderHelper.cs

```diff
@@ -141,6 +134,25 @@ public static IHostBuilder CreateHostBuilder(HostBuilderSettings settings)
                 });
         }
 
+        private static void AddJsonFileHelper(IConfigurationBuilder builder, HostBuilderResults hostBuilderResults, string filePath)
+        {
+            try
+            {
+                File.OpenRead(filePath).Dispose(); // If this succeeds, we have read permissions
+                builder.AddJsonFile(filePath, optional: true, reloadOnChange: true);
+            }
+            catch (Exception ex)
```

Comment:
I wasn't sure that we wanted to swallow everything, but we can change this if people would rather handle this differently.